ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -38,7 +38,21 @@
                     </span>
                 </span>
 
-                <!--Quick Filter Field-->
+                <!--Toggle Group State Button-->
+                <span class="btn-group">
+                    <span class="btn btn-view-nav btn-sm btn-toggle-group-state"
+                          tabindex="0" role="button"
+                          ng-click="toggleGroupState()">(
+                        <span ng-if="groupState==='collapsed'"
+                              class="group-state-nav-icon"
+                              title="Expand job groups">+</span>
+                        <span ng-if="groupState!=='collapsed'"
+                              class="group-state-nav-icon"
+                              title="Collapse job groups">-</span>
+                    )</span>
+                </span>
+
+                <!--Search Field-->
```

Comment:
This should remain `Quick Filter Field`. We're striving to refer to it everywhere as that, as we do in Help.